packages/store/src/useSearchIndex.ts

```diff
@@ -4,11 +4,14 @@ export type { SearchIndexSlice } from './types';
 
 export function useSearchIndex() {
```

Comment:
is the name still appropriate now that it returns the index and the config?

Reply:
Switched to `useSearch()` 👍